.circleci/config.yml

```diff
@@ -284,7 +284,7 @@ jobs:
               --verbose \
               -k 'not (chrome or firefox or node)' \
               --cov=pyodide_build --cov=pyodide \
-              src pyodide-build packages/micropip/
+              src/tests/ pyodide-build packages/micropip/
```

Comment:
There might be docstests in `py` that we would want to run as well I think? Generally, it's not a bad idea to run pytest on src/py. If you want to exclude things, better to do it in https://github.com/pyodide/pyodide/blob/main/setup.cfg#L9

Reply:
Okay, thanks. I just want to ignore the `_multiprocessing.py`, which of course is another point of concern.